miasm2/jitter/jitload.py

```diff
@@ -137,11 +137,7 @@ class CallbackHandlerBitflag(CallbackHandler):
 
     "Handle a list of callback with conditions on bitflag"
 
-    # Overrides CallbackHandler's implem, but do not serve for optimization
-    def has_callbacks(self, bitflag):
```

Comment:
A bit too late, but you left `has_callbacks` in the parent class `CallbackHandler`, but the implementation is inconsistent with the way `CallbackHandlerBitFlag` works. Maybe it should be removed too.